dev_tools.py

```diff
@@ -82,14 +83,32 @@ def build_webui():
     click.echo('running `npm install`')
     subprocess.check_call('npm install', cwd=cwd, shell=True)
 
-    # Build the ui
+    # Install bower packages
     click.echo('running `bower install`')
-    subprocess.check_call('bower install -p', cwd=cwd, shell=True)
+    subprocess.check_call('bower install', cwd=cwd, shell=True)
```

Comment:
@kristenmills Maybe we should switch to Yarn? https://github.com/bower/bower/issues/2298

Reply:
Bower was used on the old ui not the new one. We're using npm 5 on the new one.